certification/internal/utils/file/helper.go

```diff
@@ -1,168 +0,0 @@
-package file
```

Comment:
I didn't mean the entire file. Just the ArtifactPath and createArtifactDir

Reply:
Gotcha! Fixed.